x-pack/plugins/osquery/public/routes/live_queries/details/index.tsx

```diff
@@ -52,17 +58,34 @@ const LiveQueryDetailsPageComponent = () => {
   useLayoutEffect(() => {
     setIsLive(() => !(data?.status === 'completed'));
   }, [data?.status]);
-
-  return (
-    <WithHeaderLayout leftColumn={LeftColumn} rightColumnGrow={false}>
-      <PackQueriesStatusTable
+  const casesOwner = useMemo(() => [SECURITY_SOLUTION_OWNER], []);
```

Comment:
when we do plan to support `observability`?

Reply:
Now, just updated, thanks for bringing my attention to this 👍